example-get-started-experiments/generate.sh

```diff
@@ -116,6 +116,9 @@ dvc stage add -n evaluate \
   -M results/evaluate/metrics.json \
   --plots-no-cache results/evaluate/plots \
   python src/evaluate.py
+
+rm results/train/dvc.yaml
+git rm --cached results/train/dvc.yaml
```

Comment:
@dberenbaum I don't know what would be the best way to handle the transition to pipelines. 
Should we do this removal or instead don't include the `metrics `and `plots` sections in the root `dvc.yaml`?

Reply:
@daavoo I would prefer the latter. It seems easier to keep `Live(dvcyaml=True)` and change `dvc stage add -M results/train/metrics.json --plots-no-cache results/train/plots` to `dvc stage add -O results/train` unless I'm missing something.

I would do the same for the evaluate stage (also, why don't we evaluate the test data in the notebook?).

Reply:
> (also, why don't we evaluate the test data in the notebook?)

We do. But when converting to pipeline it gets separated into a different stage

Reply:
> change `dvc stage add -M results/train/metrics.json --plots-no-cache results/train/plots` to `dvc stage add -O results/train` unless I'm missing something

This needs to be `dvc stage add -O results/train/metrics.json -O results/train/plots` for now at least. See https://github.com/iterative/dvclive/issues/456. I still think that's better since we only need to introduce dependencies and outputs when we get to pipelines instead of special output types like metrics and plots.